tests/test_reports.py

```diff
@@ -151,7 +151,7 @@ def test_reports_sort_default(self):
             datetime.datetime(2015, 7, 5),
         ]
         dates_formatted = [isoformat(each) for each in dates]
-        reports = [
+        [
```

Comment:
Agreed. I'm taking out a bunch of these variable names, which makes `test_reports_sort` consistent with this style. This will also silence a bunch of linter errors around creating variables that are never referenced.